### ISSUE TITLE
have problem to handle urlencoded GET param

### ISSUE THREAD

**Issue by raphaelsoul** [koajs/koa#1000]
the hello world example
```
const koa  = require('koa')

const app = new koa()

app.use(ctx => {
	ctx.body = 'ok'
})

app.listen(3000)
```

the following url got result:
* http://localhost:3000/oauth/steam/return?openid.ns=http%3A%2F%2Fspecs.openid.net%2Fauth%2F2.0&openid.mode=id_res
_empty response_
* http://localhost:3000/oauth/steam/return?openid.mode=id_res _receive string 'ok'_

test environment:
* OSX 10.12.5
* node 8.0.0
* koa 2.2.0

UPDATE:
on Safari it works fine, but failed on Google Chrome. Weired.
* Chrome 59.0.3071.86 (installed by brew)
* Safari 10.1.1 (12603.2.4)

**Comment by raphaelsoul** [comment]
see also:  https://github.com/alexmingoia/koa-router/issues/360

**Comment by jonathanong** [comment]
@raphaelsoul is this still an issue with #1004?

**Comment by raphaelsoul** [comment]
Today I test it again. It works fine on Chrome. It seems to be fixed in v2.3.0. Thanks
